source/model/PadelMatch.mc

```diff
@@ -65,10 +68,24 @@ class PadelMatch {
         if (self.goldenPoint) {
             // mid game
             if (self.matchStatus.getP1Score() != 40) {
```

Comment:
at first sight, the code seems good. I just need to look carefully at this file which is the core part (unfortunately I haven't written tests for it yet, as monkey C tests are a bit different from what I'm used to 😅  #63 )

Reply:
Take your time, maybe try it, I tried inside the emulator and every time is working, even during a tie, but you know.
I want to try it live, maybe tonight

Reply:
were u able to install the app in your watch?

Reply:
Yes, tried last night!
I used sideload but there's some "bug" in saving session, because the app is working but no extra information was reported inside the report, like now. I'm not sure why, take note that I don't have uninstalled the old one and the new one seems that have overwritten the old, so there are only two possibilities:
1. My modification broke some final stats (possible)
2. Use the sideload without remove the old one make "confusion" to the watch (few possibilities)

Reply:
sorry for taking so much time. did u try any other time? the error persists?

Reply:
Yes, tried a lot of time, the session is saved but no extra information was written on it, maybe there's a cross-check with the version on the Garmin store?
So on Garmin Connect I can see the activity but like a normal Padel activity